src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -867,5 +867,42 @@ ConvSolution ConvBinWinogradRxSf2x3g1::GetSolution(const ConvolutionContext& par
     return tunable.GetSolution(params, tunable.GetPerformanceConfig(params), false);
 }
 
+bool ConvBinWinogradRxSf2x3g1Fused::IsApplicable(const ConvolutionContext&) const
+{
+    return true; // Actual checks moved to FusionMDGraph.
```

Comment:
This one looks suspicious. Might it break some usecases?

Reply:
If implementation in MDGraph is correct, then no.